Automattic/wp-calypso
# Hosting Metrics: Add tooltip to line chart

- Related to https://github.com/Automattic/dotcom-forge/issues/3436

## Proposed Changes

* Add tooltips to line charts
* Add also the time slot highlight.
* Avoid zooming in the chart with the cursor.

## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}` on an atomic site
* Move the mouse over the first line chart
* Observe the new tooltips

## Screenshots
<img width="931" alt="Screenshot 2023-08-17 at 15 02 22" src="https://github.com/Automattic/wp-calypso/assets/779993/ec24d69f-5bb8-4352-b3d1-7b721a330d0c">
<img width="939" alt="Screenshot 2023-08-17 at 15 02 40" src="https://github.com/Automattic/wp-calypso/assets/779993/083d745b-d704-427e-a979-b29630fa8a6e">
<img width="940" alt="Screenshot 2023-08-17 at 15 02 53" src="https://github.com/Automattic/wp-calypso/assets/779993/015425d6-e9f2-457f-b61d-3495780f8052">




## Screencast




https://github.com/Automattic/wp-calypso/assets/779993/198b30d2-c811-415f-bbb7-1765ab5d988a




## Pre-merge Checklist
